testsuite/src/main/resources/encoding/views/iso-8859-15.xhtml

```diff
@@ -2,7 +2,7 @@
 <html lang="de"
       xmlns:f="http://xmlns.jcp.org/jsf/core"
       xmlns:h="http://xmlns.jcp.org/jsf/html">
-    <f:view encoding="ISO-8859-15" contentType="text/html; charset=ISO-8859-15">
```

Comment:
Looks like for some reason this broke the testsuite:

```
Failed tests: 
  EncodingIT.facelets_should_allow_produces_text_html:111->checkContentType:129 expected:<text/html;charset=[iso-8859-15]> but was:<text/html;charset=[utf-8]>
  EncodingIT.facelets_should_ignore_produces_text_html_with_charset:118->checkContentType:129 expected:<text/html;charset=[iso-8859-15]> but was:<text/html;charset=[utf-8]>
  EncodingIT.facelets_should_use_content_type_view_attribute:104->checkContentType:129 expected:<text/html;charset=[iso-8859-15]> but was:<text/html;charset=[utf-8]>
Tests run: 54, Failures: 3, Errors: 0, Skipped: 4
```

Reply:
I added the encoding parameter back. Was removed by mistake...